packages/trpc/server/routers/viewer/teams/delete.handler.ts

```diff
@@ -44,4 +51,60 @@ export const deleteHandler = async ({ ctx, input }: DeleteOptions) => {
   closeComDeleteTeam(deletedTeam);
 };
 
+// cancel/delete all workflowReminders of the removed team if the realted booking doesn't belong to another active team (org teams only)
+async function deleteWorkflowRemindersOfRemovedTeam(teamId: number) {
+  const team = await prisma.team.findFirst({
+    where: {
+      id: teamId,
+    },
+  });
+
+  if (team?.parentId) {
+    const activeWorkflowsOnTeam = await prisma.workflow.findMany({
+      where: {
+        teamId: team.parentId,
+        OR: [
+          {
+            activeOnTeams: {
+              some: {
+                teamId: team.id,
+              },
+            },
+          },
+          {
+            isActiveOnAll: true,
+          },
+        ],
+      },
+      select: {
+        steps: true,
+        activeOnTeams: true,
+        isActiveOnAll: true,
+      },
+    });
+
+    for (const workflow of activeWorkflowsOnTeam) {
+      const workflowSteps = workflow.steps;
+      let remainingActiveOnIds = [];
+
+      if (workflow.isActiveOnAll) {
+        const allRemainingOrgTeams = await prisma.team.findMany({
+          where: {
+            parentId: team.parentId,
+            id: {
+              not: team.id,
+            },
+          },
+        });
+        remainingActiveOnIds = allRemainingOrgTeams.map((team) => team.id);
+      } else {
+        remainingActiveOnIds = workflow.activeOnTeams
+          .filter((activeOn) => activeOn.teamId !== team.id)
+          .map((activeOn) => activeOn.teamId);
+      }
+      deleteRemindersOfActiveOnIds([team.id], workflowSteps, true, remainingActiveOnIds);
```

Comment:
nit: It would make sense to turn this function to accept a single argument with all these arguments as props.

It usually is a good idea when there are more than 2 arguments.